src/main/resources/application.yml

```diff
@@ -10,12 +10,18 @@ worker:
   workerBaseDir: ${IEXEC_WORKER_BASE_DIR:/tmp/iexec-worker}
   gasPriceMultiplier: ${IEXEC_GAS_PRICE_MULTIPLIER:1.3} # txs will be send with networkGasPrice*gasPriceMultiplier, 4.0 means super fast
   gasPriceCap: ${IEXEC_GAS_PRICE_CAP:22000000000} #in Wei, will be used for txs if networkGasPrice*gasPriceMultiplier > gasPriceCap
-  teeEnabled: ${IEXEC_TEE_ENABLED:false}
+  teeEnabled: ${IEXEC_TEE_ENABLED:true}
   overrideBlockchainNodeAddress: ${IEXEC_WORKER_OVERRIDE_BLOCKCHAIN_NODE_ADDRESS:} #will use it if set, else will use the one given by the core
+
 wallet:
   encryptedFilePath: ${IEXEC_WORKER_WALLET_PATH:./src/main/resources/wallet/encrypted-wallet_worker1.json}
   password: ${IEXEC_WORKER_WALLET_PASSWORD:whatever}
 
+scone:
+  las:
+    host: ${IEXEC_SCONE_LAS_HOST:sgx-server.iex.ec}
```

Comment:
I guess we dont need to add the protocol since its always going to be in SSL ?

Reply:
Yes, we don't need it here.